public/scripts.js

```diff
@@ -74,12 +74,21 @@ function toggleLoader(isLoading) {
 }
 
 window.addEventListener('DOMContentLoaded', initializePage);
-
 // Function to initialize the page after the DOM has been loaded
 function initializePage() {
   initCheckboxValues();
   randomizeCredits();
   handleLocalStorage();
+  uunfocusNonemptyUsernameInput();
+}
+
+function uunfocusNonemptyUsernameInput() {
```

Comment:
Should just be `unfocusNonEmptyUsernameInput` ? Also curious why you made this change?

Reply:
Thanks, fixed.

It's a UX improvement, so if a user has already inputted their username and it's loaded in via local storage, upon reload (or when a user goes back from a collage), the page doesn't focus on the username text field, since they'd most likely want to change collage options. This is more pronounced on mobile, where the keyboard will pop up every time.

With this change, it'll only focus on the username input when it's empty, i.e., first time visitors or visits with no cache.

I had to remove the `autofocus` form the input HTML tag since that wasn't reliable, we were getting a few misses with the unfocus even with the function running on DOM load. Now it works perfectly.

Reply:
Ahh okay with mobile that makes sense. Perfect sounds good